docs/endpoints/multi_user.rst

```diff
@@ -456,6 +463,57 @@ a custom ``globus-compute-endpoint`` wrapper:
 
 (The use of ``exec`` is not critical, but keeps the process tree tidy.)
 
+
+Configuring to Accept Multiple Python Versions
+==============================================
+
+By default, Globus Compute serializes task submissions via `dill`_ with a method that
+uses Python bytecode. That's to say it does *not* serialize the source code unless asked
+to, for both technical and historical reasons, but this can sometimes cause issues.
+Because the underlying representations that Python uses for bytecode are subject to
+change at the whim of the Python developers, if the Python version running the **SDK**
+that's used to serialize and submit a task is different from the Python version of the
+**worker** that deserializes and runs the task, the worker can error (in ways that are
+often hard to debug because they happen at a low level in Python).
```

Comment:
- `s/can/may/`
- No need for the parenthesis&nbsp;&mdash;&nbsp;this is a key detail.